proto/shentu/bounty/v1/bounty.proto

```diff
@@ -47,6 +47,9 @@ message Finding {
   google.protobuf.Any encrypted_poc = 6 [(cosmos_proto.accepts_interface) = "EncryptedPoc",(gogoproto.moretags) = "yaml:\"encrypted_poc\""];
   string submitter_address  = 7 [(gogoproto.moretags) = "yaml:\"submitter_address\""];
   FindingStatus finding_status = 8 [(gogoproto.moretags) = "yaml:\"finding_status\""];
+
+  google.protobuf.Any encrypted_comment = 9 [(cosmos_proto.accepts_interface) = "EncryptedComment",(gogoproto.moretags) = "yaml:\"encrypted_comment\""];
+  google.protobuf.Any comment = 10 [(cosmos_proto.accepts_interface) = "Comment",(gogoproto.moretags) = "yaml:\"comment\""];
```

Comment:
I think comments should be stored separately

Reply:
Yes, currently only the rest query related to programs is implemented

Reply:
Your suggestion is to use a separate message to store the `comment` of `finding`?